### PR TITLE
[types/factory_girl] Make `factory` can be used named import / Add override definitions for `seq` and `sequence`

### PR DESCRIPTION
**Pull request by ishikawa**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
  - Named export https://github.com/simonexmachina/factory-girl/blob/2b71a77af25a5dab18e6849a389955e6d4a2ce72/src/index.js#L13
  - Function signature of `seq/sequence` https://github.com/simonexmachina/factory-girl/blob/2b71a77af25a5dab18e6849a389955e6d4a2ce72/src/generators/

### CODE REVIEW

File: types/factory-girl/tslint.json [hunk 1]
```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-duplicate-imports": false
```
Comment (ishikawa) [inline types/factory-girl/tslint.json:4]: I need this suppression for testing `import *` and `import {...}` in the same file.

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## factory-girl ([<kbd>unpkg</kbd>](https://unpkg.com/browse/factory-girl@latest/))
was missing the following properties:
1. ObjectAdapter
1. BookshelfAdapter
1. DefaultAdapter
1. MongooseAdapter
1. SequelizeAdapter

<details>
<summary>as well as these 1 other properties...</summary>
<p>ReduxORMAdapter</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5db412ee7fd388aaa283206b26f43fc60aeb4f37
</p>

File: types/factory-girl/tslint.json (hunk 1 above)
Comment (andrewbranch) [reply types/factory-girl/tslint.json:4]: Makes sense, but can you use a comment to disable it only where needed?

Comment (ishikawa) [reply types/factory-girl/tslint.json]: @andrewbranch Thanks for your comment. I fixed it in e8881a9.

**Review by andrewbranch — APPROVED** [review]

**Comment by ishikawa** [comment]
Ready to merge

**Pull request merged**